nx.json

```diff
@@ -0,0 +1,83 @@
+{
+  "npmScope": "hypertrace",
+  "affected": {
+    "defaultBase": "main"
+  },
+  "cli": {
+    "defaultCollection": "@nrwl/angular",
+    "packageManager": "npm"
+  },
+  "implicitDependencies": {
+    "package.json": {
+      "dependencies": "*",
+      "devDependencies": "*"
+    },
+    "package-lock.json": {
+      "dependencies": "*",
+      "devDependencies": "*"
+    },
+    ".eslintrc.json": "*",
+    "angular.json": "*",
+    "tslint.json": "*",
+    "nx.json": "*",
+    "tsconfig.base.json": "*",
+    "jest.preset.js": "*"
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": ["build", "lint", "test", "e2e"],
+        "accessToken": "YmM0N2Y0Y2QtYmQzZS00NTcwLWI3Y2ItMmRiOTgxMjllNDYxfHJlYWQ="
+      }
+    }
+  },
+  "targetDependencies": {
+    "build": [
+      {
+        "target": "build",
+        "projects": "dependencies"
+      }
+    ]
+  },
+  "generators": {
+    "@nrwl/angular:application": {
+      "style": "scss",
+      "linter": "tslint",
+      "unitTestRunner": "jest",
+      "e2eTestRunner": "cypress"
+    },
+    "@nrwl/angular:library": {
+      "style": "scss",
+      "linter": "tslint",
+      "unitTestRunner": "jest"
+    },
+    "@nrwl/angular:component": {
+      "style": "scss"
+    }
+  },
+  "defaultProject": "hypertrace-ui",
+  "projects": {
+    "assets-library": {
+      "tags": []
```

Comment:
Are we going to use tags do you think ? might be useful to automatically enforce rules between libs

Reply:
This was generated by default from nx workspace generator. I left it as it is.

Reply:
Tags are not needed to detect circular dependencies.

You wouldn't gain much from adding tags since the library projects are currently not very fine-grained. They need further splitting to achieve more gains from using Nx. This doesn't have to be done in this PR.